utils/install-open-shift-spire.sh

```diff
@@ -105,10 +106,18 @@ installSpireAgent(){
   fi
 
 # Need to copy the spire-bundle from the server namespace
-oc_cli get configmap spire-bundle -n "$SPIRESERVERPROJECT" -o yaml | sed "s/namespace: $SPIRESERVERPROJECT/namespace: $PROJECT/" | oc apply -n "$PROJECT" -f -
+oc -n "$PROJECT" get cm spire-bundle
+if [ "$?" == "0" ]; then
+  echo "WARNING: using the existing configmap spire-bundle in $PROJECT. "
+else
+  oc_cli get configmap spire-bundle -n "$SPIRESERVERPROJECT" -o yaml | sed "s/namespace: $SPIRESERVERPROJECT/namespace: $PROJECT/" | oc apply -n "$PROJECT" -f -
+fi
+
+# if the group exists, just ignore the error
+oc adm groups new "$SPIREGROUP" "$SPIRESA" 2> /dev/null
```

Comment:
assuming this file is used for both agent and server installation, and no harm in creating both anyway?

Reply:
This OpenShift group is just providing admin privileges for the service accounts used by the server and agent. No problem if created twice, for example in the same cluster.